.github/workflows/ci.yaml

```diff
@@ -38,8 +38,6 @@ jobs:
         uses: charmed-kubernetes/actions-operator@main
         with:
           provider: microk8s
-          # This is needed until https://bugs.launchpad.net/juju/+bug/1977582 is fixed
-          bootstrap-options: "--agent-version 2.9.29"
```

Comment:
Why remove this?

Reply:
this was causing CI to fail.  I believe there was a bug with libjuju that required this change.